Makefile

```diff
@@ -108,25 +115,51 @@ verify-codegen: generate
 	# TODO: Verify vendor tree is accurate
 	git diff --quiet -- ':(exclude)go.mod' ':(exclude)go.sum' ':(exclude)vendor/*'
 
-install-ginkgo:
-	go get -u github.com/onsi/ginkgo/ginkgo
-	go get -u github.com/onsi/gomega/...
-	ginkgo version
-
-install-gocov:
-	cd && GO111MODULE=on go get github.com/axw/gocov/gocov@v1.0.0
+ginkgo:
+ifeq (, $(shell which ginkgo))
+	@{ \
+	set -e ;\
```

Comment:
I think we can create a single shell-script (in `hack` directory) that receives parameters and handles `ginkgo` and `gocov` use-cases. It would be easier to maintain this script as a standalone file, and it will grant a simpler construction in `Makefile`, besides of avoiding some repetition.